core/commands/storage.go

```diff
@@ -28,6 +28,7 @@ import (
 	config "github.com/TRON-US/go-btfs-config"
 	coreiface "github.com/TRON-US/interface-go-btfs-core"
 	"github.com/TRON-US/interface-go-btfs-core/path"
+	"github.com/alecthomas/units"
```

Comment:
group the imports

Reply:
This one is actually in correct order (last group out of repo imports). I added a group for go-btfs-common imports since there are too many in this file.